README.md

```diff
@@ -1,91 +1,64 @@
-DSpot: a Test Amplification Tool for Java [![Build Status](https://travis-ci.org/STAMP-project/dspot.svg?branch=master)](https://travis-ci.org/STAMP-project/dspot)[![Coverage Status](https://coveralls.io/repos/github/STAMP-project/dspot/badge.svg?branch=master)](https://coveralls.io/github/STAMP-project/dspot?branch=master)
-=====================================================================================================================
+# DSpot
 
-### For a more step by step documentation, see the [wiki](https://github.com/STAMP-project/dspot/wiki).
+[![Build Status](https://travis-ci.org/STAMP-project/dspot.svg?branch=master)](https://travis-ci.org/STAMP-project/dspot)[![Coverage Status](https://coveralls.io/repos/github/STAMP-project/dspot/badge.svg?branch=master)](https://coveralls.io/github/STAMP-project/dspot?branch=master)
 
-### [Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
+[Riot chat room](https://riot.im/app/#/room/!vnCyWaGJbxESAkzyqh:matrix.org)
 
-### What is Dspot?
+DSpot is a tool that generates missing assertions in JUnit tests.
+DSpot take as input a Java project with an existing test suite.
+As output, DSpot outputs new test cases on console. 
+DSpot supports Java project built with Maven and Gradle project (see the `--automatic-builder` option)
```

Comment:
Java project built with Maven and Gradle project -> Java projects built with Maven and Gradle